third_party/BUILD

```diff
@@ -345,17 +345,15 @@ filegroup(
 
 # TODO(https://github.com/bazelbuild/bazel/issues/18214): After fixing Guava leak
```

Comment:
This comment should probably be removed. Has the leak mentioned in https://github.com/bazelbuild/bazel/issues/18214 been fixed?